### PR TITLE
Add VTOL tuning information and pages to QuadPlane

### PR DESCRIPTION
**Pull request by Hwurzburg**
a good deal of effort to get rid of non-applicable stuff and re-focus on QP,  and some recommended values will need to be adjusted ....but I think its close

### CODE REVIEW

File: plane/source/docs/quadplane-vtol-tuning-process.rst [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+4. Gradually increase the stick deflection to 100%
+
+Halt the tests if the aircraft overshoots level significantly or if the aircraft oscillates, the aircraft may require manual tuning (:ref:`see next section <ac_rollpitchtuning>`) before QAUTOTUNE can be run.
+
+Set :ref:`Q_A_RATE_FF_ENAB <Q_A_RATE_FF_ENAB>` to 1 after the tests are complete.
+
+Manual tuning of Roll and Pitch
+-------------------------------
+
+Manual tuning may be required to provide a stable tune before QAUTOTUNEE is run, or if QAUTOTUNEE does not produce an acceptable tune. The process below can be done on roll and pitch at the same time for a quick manual tune provided the aircraft is symmetrical. If the aircraft is not symmetrical then the process should be repeated for both roll and pitch individually.
```
Comment (tridge) [inline plane/source/docs/quadplane-vtol-tuning-process.rst:179]: spelling QAUTOTUNE

File: plane/source/docs/quadplane-vtol-tuning-process.rst [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+1. Take off in QHOVER or QSTABILIZE
+2. Hold a roll or pitch input
+3. Release the stick and observe the overshoot as the aircraft levels itself
+4. Gradually increase the stick deflection to 100%
+
+Halt the tests if the aircraft overshoots level significantly or if the aircraft oscillates, the aircraft may require manual tuning (:ref:`see next section <ac_rollpitchtuning>`) before QAUTOTUNE can be run.
+
+Set :ref:`Q_A_RATE_FF_ENAB <Q_A_RATE_FF_ENAB>` to 1 after the tests are complete.
+
+Manual tuning of Roll and Pitch
```
Comment (tridge) [inline plane/source/docs/quadplane-vtol-tuning-process.rst:176]: I think filtering should be done before D gains

File: plane/source/docs/quadplane-vtol-tuning.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+The :ref:`Q_A_ANG_YAW_P<Q_A_ANG_YAW_P>` is the gain on the error between the autopilots desired heading and actual heading which is fed into the Rate controller to demand a rotation rate. The :ref:`Q_A_RAT_YAW_P<Q_A_RAT_YAW_P>` is the gain applied to the difference between demanded rotation rate and actual.
+
+The :ref:`Q_YAW_RATE_MAX<Q_YAW_RATE_MAX>` parameter controls how quickly QuadPlane rotates based on a pilot’s yaw input in stabilized modes. 
+
+Altitude Tuning
+===============
+
+The QHOVER (altitude hold) related tuning parameters are related to the vertical position controller and the motor thrust scaling, which linearizes the throttle to motor thrust response to improve the position controllers response.
+
+TBD!
```
Comment (tridge) [inline plane/source/docs/quadplane-vtol-tuning.rst:58]: needs some work :-)

File: plane/source/docs/quadplane-vtol-tuning.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+In-flight Tuning
+================
+
+Many parameters can be tuned while in flight, see :ref:`common-transmitter-tuning`
+
+Filter tuning
+=============
+
+QuadPlanes are often affected by vibration and tuning the various software filters available is critical to achieving an overall tune.
```
Comment (tridge) [inline plane/source/docs/quadplane-vtol-tuning.rst:79]: should be moved higher, as greatly affects PID tuning

Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning.rst]: done

File: plane/source/docs/quadplane-vtol-tuning.rst (hunk 3 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning.rst]: made me go and figure it out, didnt you?....took my best shot at it

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 2 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:176]: done

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst]: fixed

File: plane/source/docs/quadplane-vtol-tuning-process.rst [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+There are several things that the pilot can do to minimize the risk during the early tuning process:
+
+1. The pilot should conduct a motor number and orientation check (see :ref:`Checking the motor numbering with the Mission Planner Motor test <connect-escs-and-motors_testing_motor_spin_directions>`). Care should be taken to ensure that the correct frame type is selected. Incorrect frame type can result in a very fast yaw rotation or complete loss of control. Take note of the output percentage required to spin the propellers and ensure that:
+
+- :ref:`Q_M_SPIN_ARM <Q_M_SPIN_ARM>` is set high enough to spin the motors cleanly.
+- :ref:`Q_M_SPIN_MIN <Q_M_SPIN_MIN>` is set high enough to spin the motors win a minimal level of thrust.
+
+2. All flights after a significant tuning change should be done in QSTABILIZE. QSTABUILIZE provides the pilot with significantly more control over the aircraft in the event that the attitude controllers are unstable.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst]: fixed

**Pull request merged**